jobs/apiserver/templates/drain-script.erb

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+RUN_DIR=/var/vcap/sys/run/apiserver
+LOG_DIR=/var/vcap/sys/log/apiserver
+LOGFILE=$LOG_DIR/drain.log
+PIDFILE=$RUN_DIR/apiserver.pid
+
+exec 3>&1
+
+exec 1>> $LOGFILE
+exec 2>> $LOGFILE
+
+if [ ! -f $PIDFILE ]; then
```

Comment:
He who does it first wins... It will show up as a conflict which would have to be resolved.